code-challenges/sort/challenges-03.test.js

```diff
@@ -0,0 +1,283 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1
+
+Write a function named sortBackwards that takes in an array of numbers and returns the same array, with the numbers sorted, highest to smallest.
+------------------------------------------------------------------------------------------------ */
+let reverse = [1, 2, 3, 4, 5];
+
+
+const sortBackwards = (arr) => arr.sort((a,b) => a - b ).reverse();
```

Comment:
Great job getting this onto one line! Instead of using .reverse(), instead of a - b, you could have done b - a. If you have any questions on how .sort() works let me know!!!